pkg/frontend/validate.go

```diff
@@ -78,26 +79,69 @@ func (f *frontend) validateOpenShiftUniqueKey(ctx context.Context, doc *api.Open
 // prevent mischief
 var rxKubernetesString = regexp.MustCompile(`(?i)^[-a-z0-9.]{0,255}$`)
 
-func validateAdminKubernetesObjectsNonCustomer(method, groupKind, namespace, name string) error {
+func validatePermittedClusterwideObjects(gvr *schema.GroupVersionResource) bool {
```

Comment:
Can we put this somewhere common so that it can generate the counterpart in https://github.com/Azure/ARO-RP/blob/master/pkg/operator/controllers/rbac/staticresources/clusterrole.yaml ?

Reply:
I have no objections to it being moved into a common/generic location, but I would rather not blow the scope of this PR any further than it already is. If you'd like to follow up with that, I'd be happy to review.